test-node.bash

```diff
@@ -38,6 +38,7 @@ l3_token_bridge=false
 batchposters=1
 devprivkey=b6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659
 l1chainid=1337
+runL2Txs=false
```

Comment:
I prefer not to add options here if we can avoid it.. already messy enough

Reply:
Done.